<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QForm1</class>
 <widget class="QMainWindow" name="QForm1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QForm1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>1</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SERIAL PORT:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>85</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>L90:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>2</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP PORT:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>21</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>115</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>L45:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>151</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>156</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>L00:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_4">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>185</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>R00:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_5">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>225</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>230</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>R45:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_6">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>265</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>270</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>R90:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_7">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>305</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>310</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BF:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_8">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>345</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>350</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BR:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND SERIAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND UDP</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>COM?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>45</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ESP01DBG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>45</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>48</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ALIVE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ALIVE: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
